packages/integration-sdk-cli/package.json

```diff
@@ -38,16 +38,13 @@
   },
   "devDependencies": {
     "@jupiterone/integration-sdk-private-test-utils": "^8.11.0",
-    "@pollyjs/adapter-node-http": "^5.1.1",
-    "@pollyjs/core": "^5.1.1",
-    "@pollyjs/persister-fs": "^5.1.1",
+    "@pollyjs/adapter-node-http": "^6.0.5",
```

Comment:
Major version changes were considered and various testing utilities have been updated. CHANGELOG can be found here: https://github.com/Netflix/pollyjs/blob/master/CHANGELOG.md#600-2021-11-30